src/UI/Component/Dropzone/File/Standard.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace ILIAS\UI\Component\Dropzone\File;
+use ILIAS\UI\Component\Button\Button;
+
+/**
+ * Interface Standard
+ *
+ * A standard file dropzone offers the possibility to upload dropped files to the server.
+ * The dropzone also displays a button to select the files manually from the hard disk.
+ *
+ * @author  nmaerchy <nm@studer-raimann.ch>
+ *
+ * @package ILIAS\UI\Component\Dropzone\File
+ */
+interface Standard extends File {
+
+	/**
+	 * Get a dropzone like this, displaying the given message in it.
+	 *
+	 * @param string $message
+	 *
+	 * @return $this
+	 */
+	public function withMessage($message);
```

Comment:
I find it useful to have the possibility to customize the message. For example, one can more clearly announce what type of files are accepted, if dealing with restricted file types. Note that the dropzone displays a default message - the same currently used in ILIAS when creating a new file object.